release-notes.md

```diff
@@ -12,27 +12,128 @@ can view active developments on GitHub at any time.
+while also empowering users with more control and flexibility to customize
+behaviors to suit your needs.
 
-The 2.0 release will also include some API changes to informational views,
-features, and other capabilities to make TimescaleDB easier to use and manage.
+What's new in TimescaleDB 2.0:
+
+- **Multi-node TimescaleDB**: Distribute hypertables across multiple nodes for
+  more scale and reliability.
+
+- **Continuous Aggregates 2.0**: Use an improved API to manually refresh data
+  or automatically set policies (more control over your data).
+
+- **User-defined Actions**: Define custom behaviors inside the database and
+  schedule them, using our job scheduling system.
+
+- **All remaining enterprise features now in Community Edition**: everything's
+  free and a more permissive Timescale License grants more rights to users.
+
+- **New and improved informational views**: Get more insight about hypertables,
+  chunks, policies, and job scheduling.
 
 ## Release Notes
 
 In this section, we will cover historical information on
 past releases and how you can learn more.
 
+## 2.0.0-rc2 (2020-10-21)
+
+This release candidate contains bugfixes since the previous release candidate.
+
+**Minor Features**
+* #2520 Support non-transactional distibuted_exec
+
+**Bugfixes**
+* #2307 Overflow handling for refresh policy with integer time
+* #2503 Remove error for correct bootstrap of data node
+* #2507 Fix validation logic when adding a new data node
+* #2510 Fix outer join qual propagation
+* #2514 Lock dimension slices when creating new chunk
+* #2515 Add if_attached argument to detach_data_node()
+* #2517 Fix member access within misaligned address in chunk_update_colstats
+* #2525 Fix index creation on hypertables with dropped columns
+* #2543 Pass correct status to lock_job
+* #2544 Assume custom time type range is same as bigint
+* #2563 Fix DecompressChunk path generation
+* #2564 Improve continuous aggregate datatype handling
+* #2568 Change use of ssl_dir GUC
+* #2571 Make errors and messages conform to style guide
+* #2577 Exclude compressed chunks from ANALYZE/VACUUM
+
+## 2.0.0-rc1 (2020-10-06)
+
+This release adds major new features and bugfixes since the 1.7.4 release.
+We deem it moderate priority for upgrading.
+
+This release adds the long-awaited support for distributed hypertables to
+TimescaleDB. With 2.0, users can create distributed hypertables across
+multiple instances of TimescaleDB, configured so that one instance serves
+as an access node and multiple others as data nodes. All queries for a
+distributed hypertable are issued to the access node, but inserted data
+and queries are pushed down across data nodes for greater scale and
+performance.
+
+This release also adds support for user-defined actions allowing users to
+define actions that are run by the TimescaleDB automation framework.
+
+In addition to these major new features, the 2.0 branch introduces _breaking_ changes
```

Comment:
I think the fact that there are breaking changes should be more emphasized, maybe with a minor headline?

Reply:
Yeah, will add to front-piece, and we'll be sure to include in 2.0 announcement.

Reply:
Actually, @ryanbooz has some pointers in his PR to the *changes* document; I think I'll highlight the breaking changes after I merge the two (i.e., in a later PR).